resources/views/airlines.blade.php

```diff
@@ -0,0 +1,311 @@
+@php
+    $title = 'Airlines';
+    $columnTitles = ['ID', 'Name', 'Description', 'Flights'];
+@endphp
```

Comment:
Usually we do not want to put PHP code into Blade, cant you pass this directly on the attributes?